src/bloomery/emit/dbt/__init__.py

```
@@ -1,10 +1,21 @@
-compatibility target" had never said compatible with *what*: the floor is where
-generic-test ``arguments`` nesting arrived, which the emitted ``schema.yml``
-uses and which dbt 1.9 rejects outright, so the range and the emitted form are
-one decision.
+**Compatible with dbt ``>=1.10.8,<2``** (RFC 0008 D22). Stated because "the
+compatibility target" had never said compatible with *what*: the emitted
+``schema.yml`` nests generic-test arguments under ``arguments``, so the range
+and the emitted form are one decision.
+
+The floor is where dbt makes that nesting the **default**, not where the
+property arrived — and that boundary is not a minor version. 1.9 rejects the
+nested form outright, and so do 1.10.0–1.10.7: through 1.10.7 it needs the
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)